src/java.desktop/unix/native/libawt_xawt/awt/gtk2_interface.c

```diff
@@ -1850,19 +1850,6 @@ static void gtk2_paint_check(WidgetType widget_type, gint synth_state,
             x, y, width, height);
 }
 
-static void gtk2_paint_diamond(WidgetType widget_type, GtkStateType state_type,
```

Comment:
I think the function pointer initialiasation of this function `fp_gtk_paint_diamond` should also be removed along with removing the function definition.

Reply:
Thanks for the hint , I removed the initialization you mentioned.

Best regards, Matthias

Reply:
I think the declaration of function pointer should also be removed. So following should be removed from line no 150
static void (*fp_gtk_paint_diamond)(GtkStyle* style, GdkWindow* window,
        GtkStateType state_type, GtkShadowType shadow_type,
        GdkRectangle* area, GtkWidget* widget, const gchar* detail,
        gint x, gint y, gint width, gint height);

Reply:
Thanks, I removed the declaration you mentioned.
Best regards, Matthias